validator/src/main.rs

```diff
@@ -116,7 +102,7 @@ async fn validate_outbound_redirect(
         .await
         .is_err()
     {
-        anyhow::bail!("timed-out ({:?}) waiting for server to be ready", timeout);
+        bail!("timed-out ({:?}) waiting for server to be ready", timeout);
```

Comment:
tiny nit: let's make these consistent
```suggestion
        bail!("timed out ({:?}) waiting for server to be ready", timeout);
```